nautobot/ipam/tests/test_filters.py

```diff
@@ -638,6 +638,13 @@ def setUpTestData(cls):
             status=statuses[0],
             namespace=cls.namespace,
         )
+        IPAddress.objects.create(
+            address="10.1.1.1/32",
+            tenant=None,
+            status=statuses[0],
+            namespace=cls.namespace,
+            nat_inside=ip0,
+        )
```

Comment:
A bit surprised that this test case doesn't already have a `generic_filter_tests` array, but you could add one after line 464 to giver coverage for the `nat_inside` filter:

```py
generic_filter_tests = (
    ["nat_inside", "nat_inside__id"],
)
```

You might also need to add another IPAddress with a non-null `nat_inside` as IIRC the test case wants to be able to find at least two unique values to filter with.

Reply:
I added the generic filter tests and discovered that the filter was actually implemented wrong. Both the test and filterset have been updated.